src/content/configuration/experiments.mdx

```diff
@@ -184,6 +185,33 @@ Detect changes to remote resources and upgrade them automatically.
 
 Enable native CSS support. Note that it's an experimental feature still under development and will be enabled by default in webpack v6, however you can track the progress on [GitHub](https://github.com/webpack/webpack/issues/14893).
 
+- Type:
+
+  - `boolean`
+  - `CssExperimentOptions`
+
+    ```ts
+    {
+      exportsOnly?: boolean;
+    }
+    ```
+
+Experimental features:
+
+- CSS Modules support: webpack will generate a unique name for each CSS class. Use the `.module.css` extension for CSS Modules.
+- <Badge text="5.87.0+" /> Style-specific fields resolution in `package.json` files:
+  webpack will look for `style` field in `package.json` files and use that if it
+  exists for imports inside CSS files.
```

Comment:
Maybe include an example for this list item?

Reply:
> Btw, did you insert many soft line breaks?

Strange. The line break gets added by prettier. Was able to remove it from one point but not the other

Reply:
> Maybe include an example for this list item?

It is a little difficult to explain that point with an example since it has to do with module resolution inside of `node_modules` folder. Will give it a try

Reply:
> Maybe include an example for this list item?

Added an example with using bootstrap since they do have that field in their package.json file https://github.com/twbs/bootstrap/blob/main/package.json#L40

Reply:
> Btw, did you insert many soft line breaks?

This is happening due to MDX component Badge. VSCode considers the file as markdown and thats just how it formats it for me 😅